src/js/base/stories/NoneFoundStories/NoneFoundBox.stories.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { NoneFoundBox } from "../NoneFound";
+import { NoneFoundBox } from "../../NoneFound";
```

Comment:
Looks good, though in the preview the icon is missing. Would be nice to have that fixed if possible.